test/specs/elements/Header/HeaderSubheader-test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import HeaderSubheader from 'src/elements/Header/HeaderSubheader'
+import * as common from 'test/specs/commonTests'
+
+describe('HeaderSubheader', () => {
+  common.rendersChildren(HeaderSubheader)
+  const wrapper = mount(<HeaderSubheader />)
+
+  it('renders a <div /> element', () => {
+    wrapper.should.have.tagName('div')
+  })
+
+  it('adds the sd-header-subheader class', () => {
+    wrapper.should.have.className('sd-header-subheader')
+  })
```

Comment:
Conformance tests require a component to be exposed at the top level API, or as a sub component, or if not exposed it must be marked private by a prefixing `_`.

The conformance tests are very generic tests that ensure that a component has a correct _meta property, handles events, handles className correctly, and a couple other minor things.

When adding:

```js
describe.only('HeaderSubheader', () => {
  common.isConformant(HeaderSubheader)
})
```

I get this output:

```shell
  Sub
    ✔ constructor name matches filename "HeaderSubheader"
    ✔ is exported as a component or sub component
    ✔ spreads user props
    ✔ handles events transparently
    _meta
      ✔ is a static object prop
      library
        ✔ is defined
        ✔ is a META.library
      name
        ✔ is defined
        ✔ matches the filename
      parent
        ✔ matches some component name
      type
        ✔ is defined
        ✔ is a META.type
    className (common)
      ✔ has the Stardust className "sd-header-subheader"
      ✔ applies user's className to root component
```

Notice the second test was able to determine that this is exported as a sub component.  That test looks at for the HeaderSubheader as a property on the top level Stardust object, but also looks on all the static properties of all the other components (sub components).